include/libxnvme_spec.h

```diff
@@ -1278,6 +1411,60 @@ struct xnvme_spec_cmd_sfeat {
 };
 XNVME_STATIC_ASSERT(sizeof(struct xnvme_spec_cmd_sfeat) == 64, "Incorrect size")
 
+/**
+ * NVMe Command Accessor for the directive send command
+ *
+ * @struct xnvme_spec_cmd_dsend
+ */
+struct xnvme_spec_cmd_dsend {
+	uint32_t cdw00_09[10]; ///< Command dword 0 to 9
```

Comment:
Am I missing something or should this be from cdw0 to cdw11 instead.
It seems 10 and 11 are missing.

The same applies to `xnvme_spec_cmd_drecv` below

Reply:
numd is actually cdw10 and doper, dtype and dspec is cdw11. The same is for xnvme_spec_cmd_drecv

Reply:
Ah okay, maybe this could be specified in the documentation or with a comment to avoid confusion in the future :)

Reply:
Yes, sure will add